### ISSUE TITLE
videos access are allowed only under valid MIT Zoom account

### ISSUE THREAD

**Issue by pangjac** [mitmath/matrixcalc#2]
Dear MITMatrix,

For all the videos, would it be possible to open access to public: hold and shared at Youtube, or at MIT open course ware ( https://ocw.mit.edu/courses/18-s096-matrix-calculus-for-machine-learning-and-beyond-january-iap-2022/pages/lecture-notes-and-readings/ )

Thank you!

**Comment by duyongqi** [comment]
Also hope so

**Comment by stevengj** [comment]
Yes, we are hoping to post the January 2023 videos to OCW.   But it will take some time to process them.

**Comment by agerlach** [comment]
Is there any updates on this? I just watched you JuliaCon presentation (fantastic) and was thinking of trying to do a group study with some co-workers. This course is very much needed. Thanks!

**Comment by HiperMaximus** [comment]
Not to be too annoying in this thread, but I second this. I just saw your JuliaCon presentation and jumped right in here but unfortunately I couldn't watch the course as it is MIT only.

**Comment by stevengj** [comment]
OCW is converging on the videos (they had to add subtitles and do other copyright stuff) and we are also converging on a set of typeset notes.  Sorry it takes a while, but progress is being made and the videos will go up (hopefully within a couple of months).
